jwtauthenticator/jwtauthenticator.py

```diff
@@ -12,30 +12,51 @@ def get(self):
         header_name = self.authenticator.header_name
         auth_header_content = self.request.headers.get(header_name, "")
         signing_certificate = self.authenticator.signing_certificate
+        secret = self.authenticator.secret
         username_claim_field = self.authenticator.username_claim_field
         audience = self.authenticator.expected_audience
+        tokenParam = self.get_argument('token', default=False)
```

Comment:
I did a fair bit of googling on JWT tokens as query parameters and searched through the JWT RFC's 6750 and 7519.  I couldn't find a suggestion for name.   The RFC you site is particular to OAuth, so I'm guessing it doesn't strictly apply here...   but, regardless it seems making the token name configurable is certainly desirable.  My question would be is 'access_token' a good default? or is that just going to be confusing in the face of OAuth? Does the OAuth spec dictate here?

After reading this discussion https://groups.google.com/forum/#!topic/api-craft/kThABXVC2KU, I'm thinking it might be better to give the header precedence over query parameter and it should be corrected to return a 400 in the case that both methods are present.